src/FsAutoComplete/LspServers/FSharpLspClient.fs

```diff
@@ -0,0 +1,98 @@
+namespace FsAutoComplete.Lsp
+
+
+// open FsAutoComplete.Lsp
```

Comment:
@baronfel should we also consider these for cleanup?

Reply:
🔥 🔥 🔥